tests/test_rados_parquet_job.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
We should work to prepare a container to run these tests in github CI eventually. I noticed that the current container is extremely large (7.5GB) any chance to slim it down a bit?